oe_whitelabel.theme

```diff
@@ -184,3 +184,61 @@ function oe_whitelabel_preprocess_media__document__default(&$variables) {
     $variables['translations'][] = $wrapper->toFileValueObject();
   }
 }
+
+/**
+ * Implements hook_element_info_alter().
+ */
+function oe_whitelabel_element_info_alter(array &$info): void {
+  $module_handler = \Drupal::service('module_handler');
+
+  if (isset($info['select']) && $module_handler->moduleExists('slim_select')) {
+    $info['select']['#process'][] = '_oe_whitelabel_process_element_select';
+  }
+}
+
+/**
+ * Process callback for applying slim_select on elements.
+ *
+ * @param array $element
+ *   The element to alter.
+ *
+ * @return array
+ *   The altered element.
+ *
+ * @see oe_whitelabel_element_info_alter
+ */
+function _oe_whitelabel_process_element_select(array &$element): array {
+  if (!$element['#multiple']) {
+    return $element;
+  }
+
+  $element['#attributes']['class'][] = 'multi-select';
```

Comment:
It's a bit weird: Now this class is only added if slim_select is enabled. Is this by intention?

Reply:
There's no point to comment old PRs that are already merged :)